.github/workflows/docs-ci.yml

```diff
@@ -5,6 +5,16 @@ on:
   push:
     branches:
       - 0.8.x
+      -
+permissions:
+  contents: read
+  pages: write
+  id-token: write
```

Comment:
why is the `id-token` permission needed?

Reply:
because of this: https://github.com/actions/deploy-pages#security-considerations

Reply:
Yes